src/GeometricFigures/Geometric.java

```diff
@@ -0,0 +1,6 @@
+package GeometricFigures;
+
+@FunctionalInterface
```

Comment:
why functional interface? it's useful for lambda syntactic. Do you use it somewhere?